FiveCalls/Pods/DropDown/DropDown/helpers/DPDKeyboardListener.swift

```diff
@@ -36,12 +36,12 @@ extension KeyboardListener {
 		NotificationCenter.default.addObserver(
 			self,
 			selector: #selector(keyboardWillShow(_:)),
-			name: NSNotification.Name.UIKeyboardWillShow,
```

Comment:
Should we maybe avoid changing stuff in  `/Pods` since we're modifying a third party library and this will just get overwritten on an update?